packages/wonder-blocks-popover/components/popover-content-core.js

```diff
@@ -0,0 +1,125 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import type {AriaProps, StyleType} from "@khanacademy/wonder-blocks-core";
+import Colors from "@khanacademy/wonder-blocks-color";
+import {View} from "@khanacademy/wonder-blocks-core";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+import {icons} from "@khanacademy/wonder-blocks-icon";
+import IconButton from "@khanacademy/wonder-blocks-icon-button";
+
+import type {Placement} from "@khanacademy/wonder-blocks-tooltip";
+
+type Props = {|
+    ...AriaProps,
+
+    /**
+     * The content to render inside the popover.
+     */
+    children: React.Node,
+
+    /**
+     * When true, the close button is shown; otherwise, the close button is not shown.
+     */
+    closeButtonVisible?: boolean,
+
+    /**
+     * Whether we should use the default light color scheme or switch to a
+     * different color scheme.
+     */
+    color: "blue" | "dark" | "light",
```

Comment:
Is `dark` the only custom color we want to allow?

Reply:
Yes for now. I checked in Webapp and so far we are only using white or darkBlue backgrounds for popovers. Maybe we can discuss this with designers in the next sync meeting.